Getting notified of unresponsive Luigi clients (#1159)
SAP/luigi: docs/navigation-parameters-reference.md

- **description**: renders all views in new frames. This setting overrides the same-domain frame reuse.
- **default**: the parameter **defaults.isolateView** is `false` by default, and you can overwrite it using the **isolateView** value on a single node level.

### defaults.pageErrorHandler
- **type**: object
- **description**: gives you the possibility to handle a situation in which Luigi Client doesn't respond. By default, it will redirect to the home page if nothing else is specified. **timeout** is required.
- **default**: the parameter **defaults.pageErrorHandler** is not specified by default, and you can overwrite it using the **pageErrorHandler** value on a single node level.
- **attributes**:
  - **timeout** amount of time after which redirection will take effect.

>   - **viewUrl** specifies the location to redirect to on the micro frontends level(URL is not changed).

Reviewer: ```suggestion
  - **viewUrl** specifies the location to redirect to on the micro frontend level (the main URL is not changed).
```

>   - **redirectPath** specifies the location to redirect to on the Luigi level(URL is changed).

Reviewer: ```suggestion
  - **redirectPath** specifies the location to redirect to on the Luigi level (the main URL is changed).
```

    }
    ```

### pageErrorHandler
- **type**: object

> - **description**: gives you the possibility to handle a situation if Luigi client doesn't respond. By default, it will redirect to the home page if nothing else is specified.

Reviewer: ```suggestion
- **description**: gives you the possibility to handle a situation in which Luigi Client doesn't respond. By default, it will redirect to the home page if nothing else is specified.
```

- **attributes**:
  - **timeout** amount of time after which redirection will take effect.

>   - **redirectPath** redirection to this location.

Reviewer: ```suggestion
  - **redirectPath** specifies the location to redirect to.
```

>   - **errorFn** use to handle different scenarios except redirection.

Reviewer: ```suggestion
  - **errorFn** used to handle different scenarios other than redirection.
```

    }
    ```

### pageErrorHandler
- **type**: object
- **description**: gives you the possibility to handle a situation in which Luigi Client doesn't respond. By default, it will redirect to the home page if nothing else is specified. **timeout** is required.
- **attributes**:
  - **timeout** amount of time after which redirection will take effect.

>   - **viewUrl** specifies the location to redirect to on the micro frontends level(URL is not changed).

Reviewer: ```suggestion
  - **viewUrl** specifies the location to redirect to on the micro frontend level (the main URL is not changed).
```

>   - **redirectPath** specifies the location to redirect to on the Luigi level(URL is changed).

Reviewer: ```suggestion
  - **redirectPath** specifies the location to redirect to on the Luigi level (the main URL is changed).
```
